packages/example-contracts/contracts/cross-chain-warriors/CrossChainWarriors.sol

```diff
@@ -71,6 +66,7 @@ contract CrossChainWarriors is
         tokenIds.increment();
         tokenIds.increment();
 
+        _safeMint(to, newWarriorId);
```

Comment:
Maybe we should prevent reentrancy at all? So in case we make minting permissioned (instead of free as it is right now), it doesn't cause any issue.

Reply:
we can but is not a must, because as an example needs to be as simple as possible so anyone can add it's own logic. And be permissioned sounds like a business logic decision.